fuse.py

```diff
@@ -483,6 +491,11 @@ def _wrapper(func, *args, **kwargs):
             print_exc()
             return -EFAULT
 
+    def _handle_nopath(self, path):
```

Comment:
I actually disagree. The purpose of this method is to handle the path argument in those fuse operations that allow a NULL path. You'll notice that it isn't called in every fuse operation, only certain ones.

A `_decode_path()` would be useful to centralize path decoding but it should only do `path.decode(self.encoding)`